File: bnim2219-spring/src/main/java/edu/bbte/idde/bnim2219/spring/controller/dto/outgoing/NoDescriptionChoreDTO.java
package edu.bbte.idde.bnim2219.spring.controller.dto.outgoing;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NoDescriptionChoreDTO {
    private Long id;
    private String title;
    private Date deadline;
    private Integer priorityLevel;
    private Boolean done;
}

File: src/controller/dto/outgoing/noDescriptionChore.dto.ts
import { Transform } from 'class-transformer';

export class NoDescriptionChoreDTO {
  id!: number;
  title!: string;
  @Transform(({ value }) => value.toISOString().split('T')[0])
  deadline!: Date;
  priorityLevel!: number;
  done!: boolean;
}
